lib/chef/provider/cron.rb

```diff
@@ -237,7 +237,11 @@ def get_crontab_entry
         newcron = ""
         newcron << "# Chef Name: #{new_resource.name}\n"
         [ :mailto, :path, :shell, :home ].each do |v|
-          newcron << "#{v.to_s.upcase}=#{@new_resource.send(v)}\n" if @new_resource.send(v)
+          if @new_resource.send(v)
+            cron_val = @new_resource.send(v)
+            cron_val = '""' if cron_val.empty?
+            newcron << "#{v.to_s.upcase}=#{cron_val}\n"
```

Comment:
The value's allowed to have spaces in it, and doesn't need quoting unless it has leading or trailing spaces that are significant (which we've never supported and no-one's ever noticed :) ).